diffs/20240124-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-pre_20240124-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-post.md

```diff
@@ -0,0 +1,88 @@
+## Reserve changes
```

Comment:
This is missing the DAI IRM change. It looks like there is an error. Probably it's trying to incorrectly use the custom IRM format after the spell is applied to compare the values in `_validateDaiInterestRateStrategy` function.